tests/test_core/test_typesbase.py

```diff
@@ -89,36 +90,35 @@ def test_complex_struct_set_and_get(self):
                     } socket;
                 } u;
             } *
-        """)
+        """, _ffi)
         _uuid = b"01234567-abcd-abcd-abcd-0123456789ab"
         o.uuid = _uuid
         o.fd = 0xfd
         o.type = 2
         o.u.file.inode = 12345
         _file_name = b"the-filename.test"
         o.u.file.name = _file_name
-        self.assertEqual(ffi.unpack(o.uuid, len(_uuid)), _uuid)
+        self.assertEqual(_ffi.unpack(o.uuid, len(_uuid)), _uuid)
         self.assertEqual(o.fd, 0xfd)
         self.assertEqual(o.type, 2)
         self.assertEqual(o.u.file.inode, 12345)
         self.assertEqual(
-            ffi.unpack(o.u.file.name, len(_file_name)),
+            _ffi.unpack(o.u.file.name, len(_file_name)),
             _file_name
         )
 
     def test_externally_provided_ffi(self):
-        ffi = cffi.FFI()
-        ffi.cdef("""
+        _ffi.cdef("""
             typedef struct _person {
                 wchar_t first_name[32];
                 wchar_t last_name[32];
             } person_t;
-        """)
-        o = typesbase.CFFICDataWrapper("person_t *", ffi)
+        """, _ffi)
```

Comment:
Should use dist.load() to retrieve an instance of ffi like we do in normal code.